client/my-sites/pages/main.jsx

```diff
@@ -135,15 +135,7 @@ class PagesMain extends Component {
 				 * Assumes users have a somewhat working clock but shouldn't be a problem if they don't.
 				 */ }
 				<Experiment
-					name={ `explat_test_aa_weekly_calypso_${ moment.utc().format( 'GGGG' ) }_week_${ moment
-						.utc()
-						.format( 'WW' ) }` }
-					defaultExperience={ null }
-					treatmentExperience={ null }
-					loadingExperience={ null }
-				/>
-				<Experiment
-					name="explat_test_aaaaa_2021_08_26_18_59"
+					name={ `explat_test_aa_weekly_calypso_${ moment.utc().format( 'GGGG' ) }_week_${ moment.utc().week() }` }
```

Comment:
The autogenereated A/A experiment profiles have names that end in `_3` instead of `_03`.